tests/unittests/libs/test_database.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from pony import orm
+
+
+@pytest.fixture
+def fake_jira_issue():
+    return {
+        "issue_key": "TEST-1234",
+        "title": "Test Issue",
+        "url": "https://jira.com",
+        "project": "TEST",
+        "severity": "blocker",
+        "status": "Open",
+        "customer_escaped": False,
+        "date_created": "2024-01-01",
+        "last_updated": "2024-01-01",
+    }
+
+
+@pytest.mark.usefixtures("temp_sqlite_db")
+@orm.db_session
+def test_database_services_entry(temp_sqlite_db):
+    test_service = temp_sqlite_db.Services(name="test-service-entry-service")
+    all_services = temp_sqlite_db.Services.select()
+    assert test_service.name in [service.name for service in all_services], "Test service not found in database."
```

Comment:
please add `test_service` to the assert message; this way when someone debugs a failure, they have all info in the log